patches/chrome-BUILD.gn.patch

```diff
@@ -29,16 +36,24 @@ index 960921acd7ebb0ba99092745e8d30bca76821f2a..b227e43ca392c5997f4bfea3e2ba2f9e
      }
  
      sources = []
-@@ -367,6 +369,8 @@ if (is_win) {
+@@ -367,9 +378,15 @@ if (is_win) {
        "app/chrome_main.cc",
        "app/chrome_main_delegate.cc",
        "app/chrome_main_delegate.h",
 +      "//brave/app/brave_main_delegate.cc",
 +      "//brave/app/brave_main_delegate.h",
      ]
  
-     output_name = "chrome"
-@@ -475,6 +479,8 @@ if (is_win) {
+-    output_name = "chrome"
```

Comment:
I thought we had most of this stuff sorted out already? We left the name as `chrome` early on to avoid other issues and renamed later I think. @AlexeyBarabash I think you did the earlier work on this?

Reply:
I tested on official/unofficial/component build with this change and and works well :)